docs/content/community/radius-artifacts.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Artifacts on Radius"
+linkTitle: "Artifacts on Radius"
+description: "Information about press, blog posts, presentation and other artifacts on Radius"
+weight: 400
+---
+
+## Microsoft Blog posts
+
+- [Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/) 
+- [Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)
+- Ongoing posts at [Radius Blog](https://blog.radapp.io)
```

Comment:
Let's make this a markdown table instead. Something like:

| Title | Source | Date |
|-------|--------|------|
| [The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)] | Microsoft Azure Blog | Oct 16, 2023 |